### PR TITLE
Shotcut version 21.06.29

### PR DESCRIPTION
**Pull request by quhxl**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/19441)

### CODE REVIEW

File: manifests/m/Meltytech/Shotcut/21.06.29/Meltytech.Shotcut.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.singleton.1.0.0.schema.json
+PackageIdentifier: Meltytech.Shotcut
+Publisher: Meltytech LLC
```
Comment (OfficialEsco) [inline manifests/m/Meltytech/Shotcut/21.06.29/Meltytech.Shotcut.yaml:3]:
```suggestion
Publisher: Meltytech, LLC
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

**Comment by quhxl** [comment]
Why this suggestion? In the other manifests it is also written without the comma. 🤷‍♂️

**Comment by OfficialEsco** [comment]
> @OfficialEsco Why this suggestion? In the other manifests it is also written without the comma. 🤷‍♂️

Either they changed it or the previous manifests are wrong, since the previous Manifest got submitted we started doing Quality Control because Publisher, PackageName and PackageVersion needs to match what Control Panel shows.

**Comment by quhxl** [comment]
Nice to know!

**Review by OfficialEsco — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**